core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -260,7 +260,7 @@ protected AbstractProject(ItemGroup parent, String name) {
         builds = buildMixIn.getRunMap();
 
         final Jenkins j = Jenkins.get();
```

Comment:
local var is no longer needed

Reply:
removed the local variable as suggested.